### PR TITLE
Fix UIKit example card reader view can not present and privacy setting

### PR DESCRIPTION
**Pull request by jonhcbs**
## Summary

change min version to 17  since lots of compile error
fix uikit example card reader view can not present

## Tested scenarios

- run demo with SampleAppUIKit target
- pair reader
- click card reader, can not present

**Fixed issue**:  
change min version to 17  since lots of compile error
fix uikit example card reader view can not present

### CODE REVIEW

**Comment by jonhcbs** [comment]
btw:  can we customize the width/height of presented card reader screen?   we want it smaller

**Comment by kaphacius** [comment]
hey @jonhcbs 

thank you for contributing!

could you please revert your commit regarding the iOS deployment targets, those changes are already in `main`.
please leave the bugfix in, greatly appreciated!

> btw: can we customize the width/height of presented card reader screen? we want it smaller

at this time this is not possible. are you running the SDK on an iPad?

**Comment by jonhcbs** [comment]
conflict fixed.  
yes, we run on ipad

File: POSSampleApp/SampleAppUIKit/Info.plist [hunk 1]
```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSBluetoothAlwaysUsageDescription</key>
+	<string>need pair reader</string>
```
Comment (kaphacius) [inline POSSampleApp/SampleAppUIKit/Info.plist:6]: can you write something like `This application requires Bluetooth to pair and connect with the card readers`

File: POSSampleApp/SampleAppUIKit/Info.plist [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+	<key>NSCameraUsageDescription</key>
+	<string>scan bar code</string>
```
Comment (kaphacius) [inline POSSampleApp/SampleAppUIKit/Info.plist:8]: can you write something like `This application requires Camera access to scan the barcode on the back of the card reader`

**Comment by kaphacius** [comment]
> conflict fixed. yes, we run on ipad

Unfortunately it is not possible to change the sizes of the present card screen at the moment. can you reach out to your AM/Adyen support so our Product team is aware of this request?

**Comment by jonhcbs** [comment]
> > conflict fixed. yes, we run on ipad
> 
> Unfortunately it is not possible to change the sizes of the present card screen at the moment. can you reach out to your AM/Adyen support so our Product team is aware of this request?

thanks, not a big issue.  will contact support if need.

**Pull request merged**